client-tools/Shell/ShellFeature.cpp

```diff
@@ -80,6 +83,11 @@ void ShellFeature::collectOptions(
 
   options->addOption("--javascript.run-main", "Execute main function.",
                      new BooleanParameter(&_runMain));
+
+  options->addOption(
```

Comment:
Isn't this startup option relevant only for maintainer mode?
If it's disabled, what does it mean? That we fetch telemetrics from the server, but don't send it out to the collecting service? That sounds pretty useless, so the question is if we can somehow use a failure point or something similar instead of a (user-visible) startup option.

Reply:
Aah, just saw that the option is now inside an `#ifdef ARANGODB_ENABLE_MAINTAINER_MODE` section.
So that's ok. But can we use something like a failure point here instead of a startup option?